README.md

```diff
@@ -0,0 +1,93 @@
+# Briefcase Android Gradle Template
+
+A [Cookiecutter](https://github.com/cookiecutter/cookiecutter/) template
+for building Python apps that will run under Android.
+
+## Using this template
+
+The easiest way to use this project is to not use it at all - at least,
+not directly. [Briefcase](https://github.com/beeware/briefcase/) is a
+tool that uses this template, rolling it out using data extracted from a
+`pyproject.toml` configuration file.
+
+However, if you *do* want use this template directly...
+
+1.  Install
+    [cookiecutter](https://github.com/cookiecutter/cookiecutter). This
+    is a tool used to bootstrap complex project templates:
+
+        $ pip install cookiecutter
+
+2.  Run `cookiecutter` on the template:
+
+        $ cookiecutter https://github.com/beeware/briefcase-android-gradle-template
+
+    This will ask you for a number of details of your application,
+    including the <span class="title-ref">name</span> of your
```

Comment:
This has picked up some stray markup - this span is a no-op, and doesn't need to be preserved.